kubeflow/common/cert-manager/kustomization.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- ./cert-manager-1-10/cert-manager/base
+- ./cert-manager/base
```

Comment:
Shouldn't it be `cert-manager/cert-manager/base` instead?

Reply:
I removed the `cert-manager-1-10`, which, besides removing the version from the folder name, also removed the unnecessary parent folder. This `kustomization.yaml` compiles and cert-manager deploys successfully.

Reply:
That makes sense, thank you Ablai!